tools/ci_build/github/linux/docker/scripts/training/ortmodule/stage1/requirements_torch1.11.0_cu10.2.txt

```diff
@@ -1,5 +1,4 @@
---pre
--f https://download.pytorch.org/whl/torch_stable.html
+--extra-index-url https://download.pytorch.org/whl/cu102

```

Comment:
Our security team has concerns on uses of "--extra-index-url". It has risks of supply chain attacks. If pytorch team publishes a fake numpy package to https://download.pytorch.org/, then we will get numpy from pytorch instead of pypi.  See https://docs.microsoft.com/en-us/nuget/concepts/security-best-practices#nuget-feeds for more information. Though the doc was written for nuget, it applies to python too.  But, to address the concern we will need to setup an Azure Devops feed first. We can do it in another PR.

Reply:
I changed this because PyTorch has updated their installation guidelines to use "--extra-index-url". Will update this once we have a clear path forward. Thanks for sharing the information though.